types/chat.ts

```diff
@@ -311,8 +312,28 @@ export interface CreatePromptParams {
     promptName: string
 }
 
-// history
+export interface ProgrammingLanguage {
+    languageName: string
+}
+
+export type InlineChatUserDecision = 'ACCEPT' | 'REJECT' | 'DISMISS' | string
 
+export interface InlineChatResultParams {
+    requestId: string
+    inputLength?: number
+    numberOfSelectedLines?: number
+    numberOfCharactersAddedFromSuggestion?: number
+    numberOfLinesAddedFromSuggestion?: number
+    numberOfCharactersDeletedFromSuggestion?: number
+    numberOfLinesDeletedFromSuggestion?: number
```

Comment:
this is a new convention that doesn't follow any existing convention in this codebase. prefixing "numberOf" in front of a number just isn't discoverable or helpful.

what is discoverable, is using a common prefix that groups things in a meaningful way. here, the grouping is "suggestion".

and, the type already mentions `number`, so there's no need to repeat that in the name.


```suggestion
    selectedLines?: number
    suggestionAddedChars?: number
    suggestionAddedLines?: number
    suggestionDeletedChars?: number
    suggestionDeletedLines?: number
```